src/rod/__main__.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import importlib.metadata
+import logging
+import sys
+
+from rod import logging as rodlogging
+
+
+def main() -> None:
+    """
+    Main function of the ROD command line interface.
+    """
+    parser = argparse.ArgumentParser(
+        prog="rod",
+        description="ROD: The ultimate Python tool for RObot Descriptions processing.",
+        usage="%(prog)s [options] file",
+    )
+
+    # Version.
+    parser.add_argument(
+        "-V",
+        "--version",
+        action="version",
+        version=f"%(prog)s {importlib.metadata.version('rod')}",
+    )
+
+    # Verbose output.
+    parser.add_argument(
+        "-vv",
+        "--verbose",
+        action="store_true",
+        help="enable verbose output.",
+    )
+
+    # File to parse.
+    parser.add_argument(
+        "-f",
+        "--file",
+        type=str,
+        help="path to the file to parse.",
+        required=False,
+    )
+
+    # Option to display the parser file attributes.
+    parser.add_argument(
+        "-s",
+        "--show",
+        action="store_true",
+        help="show the robot model attributes.",
+    )
+
+    # Option to output a URDF or SDF file.
+    parser.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        help="Output file path.",
+    )
+
+    args = parser.parse_args()
+
+    log_level = "DEBUG" if args.verbose else "INFO"
+
```

Comment:
shouldn't this be logging.DEBUG  and logging.INFO?

Reply:
Yes! Addressed in https://github.com/ami-iit/rod/pull/57/commits/4e76bca1d0f028cbc422124c36494b9d6ab56bba